### PR TITLE
Collect dynakube manifests

### PR DESCRIPTION
**Pull request by StefanHauth**
# Description

Add K8S manifests for DynaKubes to support archive.

## How can this be tested?
* Deploy operator and a DynaKube of your choice
* run the support archive subcommand
* check that the manifest for your DynaKube is part of the support archive.

## Checklist
- [x] Unit tests have been updated/added
- [x] PR is labeled accordingly
- [x] I have read and understood the [contribution guidelines](https://github.com/Dynatrace/dynatrace-operator/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: src/cmd/support_archive/resources_test.go [hunk 1]
```diff
@@ -211,28 +211,28 @@ func TestManifestCollectionFails(t *testing.T) {
 		Return(assert.AnError)
 
 	mockedApiReader.
-		On("List", toFlatInterfaceSlice(context, getExpectedObjectsList(queries[3].groupVersionKind), queries[2].filters)...).
+		On("List", toFlatInterfaceSlice(context, getExpectedObjectsList(queries[3].groupVersionKind), queries[3].filters)...).
```
Comment (chrismuellner) [inline src/cmd/support_archive/resources_test.go:214]: couldn't you just iterate over `queries` and mock `mockedApiReader.On(...)` for each of the queries?

File: test/scenarios/support_archive/support_archive_test.go [hunk 2]
```diff
@@ -29,6 +30,7 @@ import (
 	"github.com/stretchr/testify/require"
 	"golang.org/x/exp/slices"
 	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```
Comment (chrismuellner) [inline test/scenarios/support_archive/support_archive_test.go:33]:
```suggestion
	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

**Review by chrismuellner — CHANGES_REQUESTED** [review]

File: src/cmd/support_archive/resources_test.go (hunk 1 above)
Comment (StefanHauth) [reply src/cmd/support_archive/resources_test.go:214]: Mocked return values for `query[1]` and `query[2]` are different than the rest. Could iterate 3-8.

**Review by waodim — APPROVED** [review]
LGTM

**Review by chrismuellner — APPROVED** [review]

**Pull request merged**